pages/index.js

```diff
@@ -1,9 +1,11 @@
 import BooksCard from '../component/booksCard';
+import Header from '../component/header';
 
 export default function Home() {
   return (
     <>
-      <h1>My Book App </h1>
+      <Header />
+      <h1>Best Sellers</h1>
```

Comment:
```suggestion
      <h2>Best Sellers</h2>
```
There is already a `h1` inside the `header`. You should only have one `h1` for each page. Therefore (and also because of semantic/SEO reasons)  use a `h2` instead.

Reply:
fix